proto/beacon/p2p/v1/types.proto

```diff
@@ -167,16 +167,17 @@ message Crosslink {
 }
 
 message BeaconBlock {
+  // Deprecated
```

Comment:
why did you change the ordering, please revert

Reply:
i am dependent on field order for self_signed function

Reply:
reverted